ws-nextjs-app/pages/[service]/live/[id]/LivePageLayout.tsx

```diff
@@ -108,9 +109,13 @@ const LivePage = ({ pageData }: ComponentProps) => {
             )}
           </div>
           <div css={styles.secondSection}>
+            <a href="http://localhost:7081/pidgin/live/c07zr0zwjnnt?post=asset%3Aba735203-6eff-4768-83ce-74098a3ee92a#post">
```

Comment:
I imagine there is some restriction with this approach but what stops using the native scroll behaviour afforded by using `#` in the url a la this stack overflow answer: https://stackoverflow.com/a/2835166/2156186 

```suggestion
            <a href="http://localhost:7081/pidgin/live/c07zr0zwjnnt?post=asset%3Aba735203-6eff-4768-83ce-74098a3ee92a#Aba735203-6eff-4768-83ce-74098a3ee92a">
```
You need to provide the get parameter to allow the server to retrieve the correct page, if each post was given the correct `id` matching the asset id the native scroll behaviour should work (I imagine in a no js setting too)

Reply:
ah it looks like the above change may not be necessary..

They appear to add a `post` id to the linked post:
<img width="751" alt="image" src="https://github.com/bbc/simorgh/assets/9645462/d67c05a1-6ee0-4dab-88ce-f3a6574e61ec">

So as per the stackoverflow post, the `#post` should jump to the post that has been given the id; we don't need to id them individually with an asset id.

Reply:
Fab, I've implemented a basic 'backToLatest' componanent with an id of 'post' and this has solved the no-JS problem. Now look at adding more functionality.